### DISCUSSION TITLE
Animated Dial. Custom dial background, how to prevent black sweep?

### DISCUSSION THREAD

**Discussion by jacobohara77** [Bodmer/TFT_eSPI#2494] [Q&A - Sprites]
Hello, I'll begin by saying this is my first delve into any sort of custom computer program work! 
I'm trying to alter the dial to be a twin needle gauge to show the PSI in both the front and rear air bag of my classic car project. I've converted a jpeg to hexadecimal and have it set as my back ground, however when the needle sweeps, it leaves a black trace, can this be removed or made transparent?
At this time I have not altered any code from the standard animated dial...trying to ease in slowly!

My setup is a ESP32C3 with a GC9A01 screen
![dial](https://user-images.githubusercontent.com/127700439/224581382-7dcbb747-d87e-401d-bcd7-284ff2883fa8.jpeg)

**Comment by Bodmer** [comment]
Does the original animated dial sketch work?

**Reply by jacobohara77** [reply]
Yes it does.
Thank you

**Comment by RndUsername** [comment]
I have a similar problem, although I use the demo project and have white instead of black sweeps. 

![ezgif-3-e7b87b59ec](https://user-images.githubusercontent.com/39034509/227805835-9d3c2308-0fbe-4044-a604-ae53c0a5c7a2.gif)

**Reply by Bodmer** [reply]
This indicates the screen interface does not support reading back pixel colours.

You could try the following sketch which uses sprites:
https://github.com/Bodmer/TFT_eSPI/discussions/860#discussioncomment-208303
